app/services/import_polygons.rb

```diff
@@ -68,7 +67,7 @@ def location_in_scope?
 
   def get_buffers(polygons)
     buffers = {}
-    BUFFER_DISTANCES_IN_MILES.each do |distance|
+    Search::RadiusSuggestionsBuilder::RADIUS_OPTIONS.each do |distance|
```

Comment:
I just realised we need to also include legacy radius options (15 and 20) for job alerts that use these.